k8s/airflow/airflow-app.yaml

```diff
@@ -112,38 +113,38 @@ spec:
   minReplicas: 1
   maxReplicas: 3
   metrics:
-  - type: Resource
-    resource:
-      name: cpu
-      target:
-        type: Utilization
-        averageUtilization: 70
+    - type: Resource
```

Comment:
I am seeing many changes to do with indentation and etc....is this desired changes or not? @123MwanjeMike

Reply:
The indentation changes are made by the prettier pre-commit hook.
I’d classify them as desired since they keep the code consistent